themes/digital.gov/layouts/authors/list.html

```diff
@@ -9,9 +9,9 @@
           <header>
 
             {{- if .Params.github -}}
-              <img class="profile" src="https://github.com/{{- .Params.github -}}.png?size=100" srcset="https://www.github.com/{{-  .Params.github -}}.png?size=200" alt="Photo: {{- .Params.display_name -}}">
+              <img class="profile" src="https://github.com/{{- .Params.github -}}.png?size=100" srcset="https://www.github.com/{{-  .Params.github -}}.png?size=200" alt="Profile image for&nbsp;{{- .Params.display_name -}}">
```

Comment:
I don't think we need the non-breaking space here.

Reply:
On my browser, the empty space is ignored. I have removed the non-breaking space.